### PR TITLE
Instance Store

### PR DESCRIPTION
**Pull request by elibolonur**
- Do not merge yet
- Docs will be updated
- Featuering new instance and serverjson store

### CODE REVIEW

**Comment by digi604** [comment]
looks very good... I like it. Tests fail though.

**Comment by elibolonur** [comment]
Beta.13 version is corrupt at the moment as there are 2 functions that use same name on export. I've sent info to @yasatbilgin  how to solve it.

File: src/ui/serverJSONStore.js [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+        systemLang: null,
+        locale: null,
+        userfullname: null,
+        ajaxtoken: null,
+        isCustomerUI: false,
+        batteryLevel: 0,
+        statusBadge: '',
+        syncEvents: []
+    })
+
```
Comment (digi604) [inline src/ui/serverJSONStore.js:46]: can we move locale and systemLang to i18n store?
can we add SystemStore for stuff like batteryLevel etc?
can we add FileStore for files, categories etc?

just splitting this up so we do not have 1 huge store for everything?

**Review by digi604 — CHANGES_REQUESTED** [review]

**Comment by digi604** [comment]
Is there any way to test at least some of the stuff? -3% coverage.... is bad

File: src/db/query.js [hunk 2]
```diff
@@ -30,8 +30,7 @@ function hasCached(query) {
     )
 }
 
-async function query(query, db = null, removeNull = false) {
-    console.log(query)
```
Comment (digi604) [inline src/db/query.js:34]: can we make that configurable with a env variable if this should print?

**Pull request closed without merging**